libs/sql-schema-describer/src/postgres.rs

```diff
@@ -57,6 +57,36 @@ impl<'a> super::SqlSchemaDescriberBackend for SqlSchemaDescriber<'a> {
         let table_names = self.get_table_names(schema).await?;
         let mut tables = Vec::with_capacity(table_names.len());
 
+        if self.is_cockroach() {
+            // Currently, we ignore all hidden columns from CockroachDB.
+            // However, these still show up from get_indices, where CockroachDB can place implicit
+            // columns in front of indexes and primary keys.
+            // For now, remove all hidden columns from the indexes and PK, removing them as an index
+            // or PK if every column is implicit.
+            for (table_name, table_indexes) in indexes.iter_mut() {
+                let mut table_columns = HashSet::new();
+                match columns.get(table_name) {
```

Comment:
i had a lot of trouble generating the rust to write this, so if there is a more rust-matic way of writing this lmk.